src/main/java/com/minecolonies/coremod/colony/managers/EventManager.java

```diff
@@ -53,6 +59,11 @@
      */
     private final Map<Integer, IColonyEvent> events = new HashMap<>();
 
+    /**
+     * The event descriptions of this colony.
+     */
+    private final LinkedList<IColonyEventDescription> eventDescs = new LinkedList<>();
+
```

Comment:
ColonyEvents are quite different from event descriptions, those are ongoing timed events for a colony like a pirate raid. I don't think that the event descriptions belong into here at all, better move them to their own file.

Reply:
I have them here, because that way i can reuse code from the EventManager, but i can move them to their own class.

Reply:
ye make the own manager for it, like ColonyHistoricEventManager or os

Reply:
Maybe call it ColonyLogManager?

Reply:
it's kinda a log

Reply:
I called it EventDescriptionManager, but i want to test it before pushing it, and i can't do that because of the crash.

Reply:
this should be done now.